src/Chrome.php

```diff
@@ -48,7 +48,7 @@ public function __construct($binary = 'chrome-pdf', $processClass = null)
      */
     public function createProcess($cmdline)
     {
-        return new $this->{'processClass'}($cmdline);
+        return new $this->{'processClass'}([$cmdline]);
```

Comment:
there's a docblock issue here, `createProcess` is doc'd as receiving a string, but actually [receives an array](https://github.com/SynergiTech/chrome-pdf-php/blob/2dc7b9f7a914f8c9d6307eba5bbc54f738e6c2bf/src/Chrome.php#L212)
as such, we don't need to wrap it